license_manager/apps/api/tasks.py

```diff
@@ -190,31 +128,30 @@ def link_learners_to_enterprise_task(learner_emails, enterprise_customer_uuid):
         )
 
 
-@shared_task(base=LoggedTaskWithRetry, soft_time_limit=SOFT_TIME_LIMIT, time_limit=MAX_TIME_LIMIT)
-def send_reminder_email_task(custom_template_text, email_recipient_list, subscription_uuid):
+def _batch_notify_or_remind_assigned_emails(
+    custom_template_text,
+    email_recipient_list,
+    subscription_uuid,
+    allowed_batch_size,
+    campaign_uuid,
+    action_type,
+):
     """
-    Sends license activation reminder email(s) asynchronously.
-
-    Arguments:
-        custom_template_text (dict): Dictionary containing `greeting` and `closing` keys to be used for customizing
-            the email template.
-        email_recipient_list (list of str): List of recipients to send the emails to.
-        subscription_uuid (str): UUID (string representation) of the subscription that the recipients are associated
-            with or will be associated with.
-    Raises:
-      An exception if the number of assigned licenses to send reminders for exceeds the max
-      allowed request size of the Braze campaign endpoint.
+    Does the work of sending notification and reminder emails
```

Comment:
nit: perhaps add a mention that the "notification" and "reminder" are related to the `action_type` argument so it's slightly more evident what `action_type` is referring to directly.

```suggestion
    Does the work of sending notification and reminder emails (depending on the supplied `action_type`)
    for assigned licenses.
```

Reply:
Maybe I'll follow best practices and document the parameters in the docstring! :p